src/DDserver.cpp

```diff
@@ -50,7 +52,7 @@ namespace DoubleD
         return true;
     }
 
-    void DDserver::m_handlePrefixes(char* _argv[], int _argc)
+    void DDserver::m_handleCommandLineArguments(char* _argv[], int _argc)
```

Comment:
I think 'prefix' is a really uncommon name!

Reply:
Other peoole probably don't know what you mean when you say 'prefix'. Command-line argument/flag really is the appropriate term here.